pkg/command/storageeos.go

```diff
@@ -108,24 +108,6 @@ func StorageEOS(cfg *config.Config) *cli.Command {
 										"enable_home":            cfg.Reva.Storages.EOS.EnableHome,
 										"user_layout":            cfg.Reva.Storages.EOS.Layout,
 									},
-									"local": map[string]interface{}{
```

Comment:
getting rid of unnecesarry config options is ok: https://github.com/owncloud/ocis-reva/issues/155